src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -127,7 +127,7 @@ public AutoFlushingStream(Stream inner)
             public override void Write(byte[] buffer, int offset, int count)
             {
                 _inner.Write(buffer, offset, count);
-                _inner.Flush();
+                _inner.FlushAsync();
```

Comment:
@NTaylorMullen didn't run into any more crashes after adding this, so I assume its' the fix we're going with? Is there an issue associated with it?

Reply:
Ya lets use this. Could you update the `Flush` override above to call into `FlushAsync` as well?

No issue for it.